gpio/README.md

```diff
@@ -103,16 +103,16 @@ Connection = openHAB
 Pin = 17
 PUD = UP
 EventDetection = BOTH
-Destination = back-door
+Destination = back_door
 Level = DEBUG
 
 [Sensor2]
 Class = gpio.rpi_gpio.RpiGpioSensor
 Connection = openHAB
 Poll = 1
-Pin = 17
+Pin = 18
```

Comment:
In this config example are two sensors, both using pin 17. I think it is not the standard use case to have several sensors to read from the same pin, would it even work with RPi.GPIO?
To me this example was quite confusing since it told me I need to sensors for one pin. To make them separate I changed the pin of the second sensor.